pkg/grapicmd/context.go

```diff
@@ -129,8 +121,8 @@ var CtxSet = wire.NewSet(
 func ProvideFS(c *Ctx) afero.Fs                 { return c.FS }
 func ProvideViper(c *Ctx) *viper.Viper          { return c.Viper }
 func ProvideExecer(c *Ctx) exec.Interface       { return c.Execer }
-func ProvideIO(c *Ctx) *cli.IO                  { return c.IO }
+func ProvideIO(c *Ctx) clib.IO                  { return c.IO }
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideIO should have comment or be unexported